src/redux/modules/pastelDB/pastelDBSlice.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
Please move this to `src/features/<your feature>`

Reply:
Great! Thanks